src/main/java/com/gw/jpa/Environment.java

```diff
@@ -22,11 +17,11 @@ public class Environment {
 
 	private String name, type, bin, pyenv, basedir;
 
-	@ManyToOne(fetch = FetchType.LAZY)
+	@ManyToOne(fetch = FetchType.EAGER)
 	@JoinColumn(name = "hostid")
 	@JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
 	private Host hostobj;
 
-	@Column(columnDefinition = "LONGTEXT")
+	@Lob
```

Comment:
Will this work with existing records as I mentioned? Or do people need to do db migration first?

Reply:
It works, tested manually, no need for migration. Need more testing though